src/System.Data.SqlClient/src/System/Data/Common/AdapterUtil.cs

```diff
@@ -1142,5 +1142,21 @@ internal static bool IsAzureSqlServerEndpoint(string dataSource)
 
             return false;
         }
+
+        static internal ArgumentOutOfRangeException InvalidDataRowVersion(DataRowVersion value)
+        {
+#if DEBUG
```

Comment:
Interesting. @jaredpar, why wouldn't Roslyn optimize away empty branches of switches?